go.mod

```diff
@@ -33,21 +33,22 @@ require (
 	github.com/google/go-cmp v0.3.1
 	github.com/google/shlex v0.0.0-20150127133951-6f45313302b9
 	github.com/google/uuid v1.1.1 // indirect
+	github.com/gorilla/mux v1.7.4 // indirect
 	github.com/grpc-ecosystem/go-grpc-middleware v1.2.0
 	github.com/grpc-ecosystem/grpc-opentracing v0.0.0-20180507213350-8e809c8a8645
 	github.com/hashicorp/go-immutable-radix v1.0.0
 	github.com/hashicorp/golang-lru v0.5.1
 	github.com/hashicorp/uuid v0.0.0-20160311170451-ebb0a03e909c // indirect
-	github.com/imdario/mergo v0.3.7 // indirect
+	github.com/imdario/mergo v0.3.9 // indirect
 	github.com/ishidawataru/sctp v0.0.0-20191218070446-00ab2ac2db07 // indirect
 	github.com/jaguilar/vt100 v0.0.0-20150826170717-2703a27b14ea
 	github.com/mitchellh/hashstructure v1.0.0
 	github.com/morikuni/aec v0.0.0-20170113033406-39771216ff4c
 	github.com/opencontainers/go-digest v1.0.0-rc1
 	github.com/opencontainers/image-spec v1.0.1
-	github.com/opencontainers/runc v1.0.0-rc9.0.20200221051241-688cf6d43cc4
-	github.com/opencontainers/runtime-spec v1.0.1
-	github.com/opencontainers/selinux v1.3.2 // indirect
+	github.com/opencontainers/runc v1.0.0-rc9.0.20200102164712-2b52db75279c
```

Comment:
https://github.com/containerd/containerd/blob/04985039cede6aafbb7dfb3206c9c4d04e2f924d/vendor.conf#L34

Reply:
Interesting; I thought I changed that one; let me check.

Reply:
Yes, go modules doesn't accept v1.0.0-rc10; it reverts it to `v1.0.0-rc9.0.20200102164712-2b52db75279c` 😞

Reply:
Even if you just add a commit and let it figure out the prefix? What other dependency causes it to switch then? Otherwise, we need a replace rule for this as well then. 😞

Reply:
Ah, I guess go.mod just can't sort this. So find out what depends on `rc9` and if we can't change that then add a replace rule.

Reply:
Ugh yes; let me try find what dependency causes it

Reply:
Looks like it's github.com/AkihiroSuda/containerd-fuse-overlayfs

Reply:
Ok, replace rule it is then. If there is no actual code change between them you can leave `rc9` to the require section for people who forgot to replace. Maybe makes sense to notify `runc` repo as well that their tagging is not compatible with `go.mod` sorting (although I wouldn't say this is their fault).

Reply:
Opened https://github.com/AkihiroSuda/containerd-fuse-overlayfs/pull/13 in the meantime

Reply:
Looks like there's no diff in the vendored code;

```patch
diff --git a/go.mod b/go.mod
index 5235bbab..d5d83a0c 100644
--- a/go.mod
+++ b/go.mod
@@ -81,3 +81,6 @@ replace github.com/jaguilar/vt100 => github.com/tonistiigi/vt100 v0.0.0-20190402
 replace github.com/containerd/containerd => github.com/containerd/containerd v1.3.1-0.20200511142502-04985039cede

 replace github.com/docker/docker => github.com/docker/docker v17.12.0-ce-rc1.0.20200310163718-4634ce647cf2+incompatible
+
+// TODO: remove after https://github.com/moby/buildkit/pull/1482 is merged and vendored
+replace github.com/opencontainers/runc => github.com/opencontainers/runc v1.0.0-rc10
diff --git a/go.sum b/go.sum
index f27d8611..f1fbdcba 100644
--- a/go.sum
+++ b/go.sum
@@ -164,10 +164,8 @@ github.com/opencontainers/go-digest v1.0.0-rc1 h1:WzifXhOVOEOuFYOJAW6aQqW0TooG2i
 github.com/opencontainers/go-digest v1.0.0-rc1/go.mod h1:cMLVZDEM3+U2I4VmLI6N8jQYUd2OVphdqWwCJHrFt2s=
 github.com/opencontainers/image-spec v1.0.1 h1:JMemWkRwHx4Zj+fVxWoMCFm/8sYGGrUVojFA6h/TRcI=
 github.com/opencontainers/image-spec v1.0.1/go.mod h1:BtxoFyWECRxE4U/7sNtV5W15zMzWCbyJoFRP3s7yZA0=
-github.com/opencontainers/runc v0.0.0-20190115041553-12f6a991201f/go.mod h1:qT5XzbpPznkRYVz/mWwUaVBUv2rmF59PVA73FjuZG0U=
-github.com/opencontainers/runc v1.0.0-rc6/go.mod h1:qT5XzbpPznkRYVz/mWwUaVBUv2rmF59PVA73FjuZG0U=
-github.com/opencontainers/runc v1.0.0-rc9.0.20200102164712-2b52db75279c h1:9EMFehIYZPnCFOz8NJ5d3DyBUY51q/G91WsGBK304jY=
-github.com/opencontainers/runc v1.0.0-rc9.0.20200102164712-2b52db75279c/go.mod h1:qT5XzbpPznkRYVz/mWwUaVBUv2rmF59PVA73FjuZG0U=
+github.com/opencontainers/runc v1.0.0-rc10 h1:AbmCEuSZXVflng0/cboQkpdEOeBsPMjz6tmq4Pv8MZw=
+github.com/opencontainers/runc v1.0.0-rc10/go.mod h1:qT5XzbpPznkRYVz/mWwUaVBUv2rmF59PVA73FjuZG0U=
 github.com/opencontainers/runtime-spec v0.1.2-0.20190507144316-5b71a03e2700/go.mod h1:jwyrGlmzljRJv/Fgzds9SsS/C5hL+LL3ko9hs6T5lQ0=
 github.com/opencontainers/runtime-spec v1.0.1 h1:wY4pOY8fBdSIvs9+IDHC55thBuEulhzfSgKeC1yFvzQ=
 github.com/opencontainers/runtime-spec v1.0.1/go.mod h1:jwyrGlmzljRJv/Fgzds9SsS/C5hL+LL3ko9hs6T5lQ0=
diff --git a/vendor/modules.txt b/vendor/modules.txt
index cada10c1..190fece9 100644
--- a/vendor/modules.txt
+++ b/vendor/modules.txt
@@ -257,7 +257,7 @@ github.com/opencontainers/go-digest
 github.com/opencontainers/image-spec/identity
 github.com/opencontainers/image-spec/specs-go
 github.com/opencontainers/image-spec/specs-go/v1
-# github.com/opencontainers/runc v1.0.0-rc9.0.20200102164712-2b52db75279c
+# github.com/opencontainers/runc v1.0.0-rc9.0.20200102164712-2b52db75279c => github.com/opencontainers/runc v1.0.0-rc10
 github.com/opencontainers/runc/libcontainer/cgroups
 github.com/opencontainers/runc/libcontainer/configs
 github.com/opencontainers/runc/libcontainer/system
```